cobra/util/optlang.py

```diff
@@ -0,0 +1,46 @@
+def split_bounds(lower_bound, upper_bound):
```

Comment:
I do use make_forward_and_reverse_bounds (split_bounds) in cobra.core.Model, so having that separated made some sense. I can certainly move the other over, but it seemed odd to have it floating in `Reaction.py`.
Happy to do whichever though